common/eth2_network_config/src/lib.rs

```diff
@@ -149,10 +144,8 @@ impl Eth2NetworkConfig {
                     )
                 })
         } else {
-            self.genesis_state::<E>(genesis_state_url, timeout, log)?
-                .map(|state| state.genesis_validators_root())
-                .map(Result::Ok)
-                .transpose()
+            self.get_genesis_state_from_bytes::<E>()
```

Comment:
Switching to the new sync `get_genesis_state_from_bytes` method, as we don't need to download genesis state and `genesis_state` method is now async.